source/JsMaterialX/JsMaterialXFormat/post.js

```diff
@@ -451,5 +466,47 @@
                 return _readFromXmlString(doc, result.data, searchPath, readOptions, filesLoaded, result.filePath);
             });
         };
+
+        // Write a document to a file.
+        Module.writeToXmlFile = function(doc, fileName, writeOptions = null) {
```

Comment:
This two functions are exactly the same except a word, can we rewrite it?
Maybe one function for checking the arguments